### ISSUE TITLE
chore: remove orphaned directory from ja

### ISSUE THREAD

**Issue by schalkneethling** [mdn/translated-content#8681]
When looking for the number of HTML files still existing for the various locales, I found ~10 in `ja`. Turns out those were directories and not files. I had a conversation with @Rumyra. @dipikabh and @bsmth and we discovered that these folders no longer exist. As it also does not exist for any of the other locales, it is best to clean this up for `ja`.

**Comment by schalkneethling** [comment]
I have noticed that some other locales also have the `orphaned` folder. They do not all have all of the same content, though. This makes me wonder whether there is a reason these are kept. I am happy to clean this all up, but I thought I would ask before doing the work. @mdn/localization-team-leads Thoughts?

**Comment by yin1999** [comment]
The `orphaned` folder is used by [`sync-translated-content` workflow](https://github.com/mdn/translated-content/blob/main/.github/workflows/sync-translated-content.yml) which will move the deleted page (in `en-US`) to this folder, so we can be notified those documents have been removed (from `en-US`), and we should do something to **sync** with the `mdn/content` (deal with url redirect and something else with removing those file).

See also: https://github.com/mdn/yari/blob/2a8a572e7e97b4c8798d5db3347d7cb9227c68ce/build/sync-translated-content.ts#L171-L216

**Comment by schalkneethling** [comment]
Thank you for the infomation @yin1999 that is very good to know. It does make me wonder, do you all need some help deleting and redirecting these pages. It seems `ja` has a lot in the orphaned folder for example.

**Comment by yin1999** [comment]
> do you all need some help deleting and redirecting these pages

We can use `yarn content delete` with `--redirect` option to deal with this. But how to deal with `conflicting` and `orphaned` folders is not mentioned in the guidelines (use which command to delete those pages, updating the existing uri to the new page (maybe we should use `git log --` command to track the commits/PRs in `mdn/content`), etc.).

**Comment by mfuji09** [comment]
@schalkneethling 
You can be noticed that we are removing files in those directories little by little,  by seeing the history.

However, it needs a lot of time to be cleaned because:
Before MDN Web Docs migrated to Yari, the translation into Japanese was fairly advanced (perhaps more so than other languages). However, there were many files that for some reason no longer corresponded to the English version, and these are included in orphaned.
Even after migrating to Yari, some Japanese files were sometimes moved to orphaned by some accident when the English version was moved.
We are restoring translations using the articles in orphaned as needed while we translate and revise existing articles.

If we should remove the articles in orphaned sooner rather than later, we will hasten to review these articles. However, I think it would be more constructive to spend more time on correcting the existing articles.

**Comment by SphinxKnight** [comment]
I guess there's no deadline for cleaning those as both `orphaned` and `conflicting` are recreated/added/filled whenever there's a move / suppression in `mdn/content`.

I second @mfuji09 about current article maintenance and translation: this might not be "visible" in terms of "chores" (whether it's md conversion, conflicting/orphaned cleanup, macro removal, linting), but is still the most important action for the localization teams with a great impact on the content readers are using.

**Comment by schalkneethling** [comment]
> If we should remove the articles in orphaned sooner rather than later, we will hasten to review these articles. However, I think it would be more constructive to spend more time on correcting the existing articles.

Nope, no rush at all. I had a completely different understanding of the intent of the folder :+1: Do we have this documented somewhere? It would be good to document it if we do not so I can make sure to bring it to everyone else's attention. This thread goes a long way to document it, so this is a start. Thank you for all the feedback and clarification.

**Comment by hochan222** [comment]
> Do we have this documented somewhere?

This document does not currently exist. It would be nice to make a guideline!

**Comment by SphinxKnight** [comment]
I've committed https://github.com/mdn/translated-content/pull/9199 with my understanding and how I'd suggest to deal with those. Please comment in the PR so that we can improve this guide :)

**Comment by bsmth** [comment]
I think "removing orphaned files" is an ongoing effort. Documentation for dealing with them is added thanks to Julien in #9199:

https://github.com/mdn/translated-content/blob/main/docs/orphaned_and_conflicting.md

In that case I would close for now, but feel free to reopen if you think I've made a mistake. Thanks, all :)
